test/duration_test.rb

```diff
@@ -0,0 +1,32 @@
+require_relative "test_helper"
+
+describe HotelBooking::Duration do
+  describe "duration test" do
+    it "raise exception when an invalid date range is provided" do
+      expect {
+        HotelBooking::Duration.new(
+          start_date: "2019-02-01",
+          end_date: "2019-01-01",
+        )
+      }.must_raise ArgumentError
+    end
+  end
+  describe "" do
```

Comment:
Don't forget to add a description to your `describe` blocks. If you can't find a good name, then maybe you need to think about how you're organizing your code